server/container/guice/cassandra-rabbitmq-guice/src/test/resources/modules/mailbox/cassandra.properties

```diff
@@ -0,0 +1,12 @@
+mailbox.max.retry.acl=1
```

Comment:
The values here do not look really production ready.

This file was used to some non-related testing. Please remove it.